kannon/master.py

```diff
@@ -49,16 +49,17 @@ def build(self, root_task: gokart.TaskOnKart) -> None:
         task_queue = self._create_task_queue(root_task)
 
         # consume task queue
-        launched_task_ids: Set[str] = set()
```

Comment:
`launched_task_ids` can be replaced with `self.task_id_to_job_name`, so removed it.